README.md

```diff
@@ -7,9 +7,91 @@ Table of Contents
 =================
 
 * [Name](#name)
+* [Synopsis](#synopsis)
 * [Description](#description)
 * [Copyright & License](#copyright--license)
 
+Synopsis
+========
+
+Basic usage
+
+```
+
+# assignment in opsboy file scope.
+foo = bar; # Opsboy's assignment syntax. variable can be interpolated when startwith dollar sign and within strings.
```

Comment:
This comment has English grammar errors. Should be

"variables can be interpolated like in shell"